libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -79,6 +84,7 @@ + (instancetype)sharedManager
     static SalesforceSDKManager *sdkManager = nil;
     dispatch_once(&pred , ^{
         uid = [[[UIDevice currentDevice] identifierForVendor] UUIDString];
+        features = [NSMutableSet setWithObjects:kSFAppFeatureOAuth, nil];
```

Comment:
Agreed.  The OAuth setting should follow the same pattern that SmartStore does, as a consumer.